db/dump.py

```diff
@@ -238,6 +241,56 @@ def generate_where(row_name, start_t=start_time, end_t=end_time):
         connection.close()
 
 
+def update_sequence(seq_name, table_name):
+    """ Update the specified sequence's value to the maximum value of ID in the table.
+
+    Args:
+        seq_name (str): the name of the sequence to be updated.
+        table_name (str): the name of the table from which the maximum value is to be retrieved
+    """
+    with db.engine.connect() as connection:
+        connection.execute(text("""
+            SELECT setval('{seq_name}', max(id))
+              FROM {table_name}
+        """.format(seq_name=seq_name, table_name=table_name)))
+
+
+def update_sequences():
+    """ Update all sequences to the maximum value of id in the table.
+    """
+    # lowlevel_id_seq
+    logging.info('Updating lowlevel_id_seq...')
+    update_sequence('lowlevel_id_seq', 'lowlevel')
+
+    # highlevel_model_id_seq
+    logging.info('Updating highlevel_model_id_seq...')
+    update_sequence('highlevel_model_id_seq', 'highlevel')
+
+    # version_id_seq
+    logging.info('Updating version_id_seq...')
+    update_sequence('version_id_seq', 'version')
+
+    # model_id_seq
+    logging.info('Updating model_id_seq...')
+    update_sequence('model_id_seq', 'model')
+
+    # incremental_dumps_id_seq
+    logging.info('Updating incremental_dumps_id_seq...')
+    update_sequence('incremental_dumps_id_seq', 'incremental_dumps')
+
+    # user_id_seq
+    logging.info('Updating user_id_seq...')
+    update_sequence('user_id_seq', '"user"')
+
+    # dataset_class_id_seq
+    logging.info('Updating dataset_class_id_seq...')
+    update_sequence('dataset_class_id_seq', 'dataset_class')
+
+    # dataset_eval_sets_id_seq
+    logging.info('Updating dataset_eval_sets_id_seq...')
+    update_sequence('dataset_eval_sets_id_seq', 'dataset_eval_sets')
```

Comment:
Do we have a way of dumping production/private tables yet (e.g. "user"?). If we eventually add it then we'll need to add those tables somewhere too - here or a new method?

Reply:
We don't have a private dump yet, but I updated those sequences in this function too.